### DISCUSSION TITLE
flair support

### DISCUSSION THREAD

**Discussion by ghost** [Serene-Arc/bulk-downloader-for-reddit#822] [Ideas]
is there any way to add support for downloading posts of a particular flair from a subreddit? that would really be great

**Comment by Fakeaccount12312** [comment]
Of course there is! Just use the search feature:
```
python -m bdfr download "directory" -s subreddit -t time --search 'flair_name:"Your flairname"'
```

**Reply by ghost** [reply]
that's unfortunately not working for me. the program's being completed without a single download.

**Reply by Fakeaccount12312** [reply]
Thats strange, I tested it and it works for me. What is your exact command, so I can test if it works for me?

**Reply by ghost** [reply]
i wrote
`bdfr download "C:\K" -s fuckcars --search 'flair_name:"Satire"'`

**Reply by Fakeaccount12312** [reply]
Works for me, no clue what your issue is. The only thing I changed was the path bc my folders are structured diferent. Either that or it is a problem with the shell you use, mine is the git bash on Windows.

**Reply by Serene-Arc** [reply]
I'd need to see the full logs, or at least the first 50 lines or so when the BDFR goes through the sources.
